arm/Microsoft.MachineLearningServices/workspaces/.parameters/parameters.json

```
@@ -17,6 +17,34 @@
         "associatedApplicationInsightsResourceId": {
             "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Insights/components/adp-<<namePrefix>>-az-appi-x-001"
         },
+        "systemAssignedIdentity": {
+            "value": true
+        },
+        "computes": {
```

Lets add it to the deployment validation too:

```suggestion
        },
        "userAssignedIdentities": {
            "value": {
                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
            }
        },
        "computes": {
```

---

@MariusStorhaug Is updated.